scripts/create-release

```diff
@@ -57,6 +57,9 @@ fi
 
 # Copy the unreleased changelog directory.
 if [ -d "${source_dir}/changelog/unreleased" ]; then
+  for entry in ${source_dir}/changelog/unreleased/**/*.md; do
```

Comment:
Actually I missed this one; I think `source_dir` must be quoted.

Reply:
Nice catch, fixed.